lib/send.js

```diff
@@ -130,12 +130,17 @@ function normalizeOptions (options) {
 
   const maxage = normalizeMaxAge(options.maxAge ?? options.maxage)
 
+  const maxChunkSize = options.acceptRanges !== undefined
+    ? Number(options.maxChunkSize)
+    : null
+
   const root = options.root
     ? resolve(options.root)
     : null
 
   return {
     acceptRanges,
+    maxChunkSize,
```

Comment:
Sorry @ledyba, could you also move this to be in alphabetical order as well please?

Reply:
Thanks, I renamed to `highWatarmark` and move it in alphabetical order.